src/ieee80211_station.erl

```diff
@@ -860,6 +860,7 @@ aaa_association(State = #state{mac = MAC, data_channel_address = WTPDataChannelA
 				wtp_id = WtpId, wtp_session_id = WtpSessionId}) ->
     MACStr = format_mac(MAC),
     SessionData0 = [{'Accouting-Update-Fun', fun accounting_update/2},
+                    {'AAA-Application-Id', capwap_station},
```

Comment:
is this really the only place where AAA is used in the context of the station?

Reply:
I write application_id in ergw_aaa session. It's enough